From f261b2f1bc16c32a55a22e28bf1b2f15ac6c7a88 Mon Sep 17 00:00:00 2001
From: Kai Kang <kai.kang@windriver.com>
Date: Wed, 6 Dec 2023 10:24:46 +0800
Subject: [PATCH] configure.ac: check egrep

The macro AC_HEADER_STDC which requires AC_PROG_EGREP has been removed
from configure.ac, then it fails to substitute '@EGREP@':

|     [GEN] command-lines-input.h
| /bin/bash: line 2: @EGREP@: command not found
|     [GEN] command-count.h

Call AC_PROG_EGREP to fix the error.

Upstream-Status: Submitted [https://gitlab.com/lvmteam/lvm2/-/merge_requests/4]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index c2dbf3c9f..d9e7964c3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -88,6 +88,7 @@ AC_PROG_EGREP
 AC_PROG_MAKE_SET
 AC_PROG_MKDIR_P
 AC_PROG_RANLIB
+AC_PROG_EGREP
 AC_CHECK_TOOL([READELF], [readelf])
 AC_CHECK_TOOL(AR, ar)
 AC_PATH_TOOL(CFLOW_CMD, cflow)
